python/paddle/fluid/distributed_spliter.py

```diff
@@ -13,38 +13,63 @@
 # limitations under the License.
 
 
-def hash_name(varlist, pserver_endpoints):
+class DistributedSpliter(object):
```

Comment:
"Distributed" seems very general, consider rename to PSStrategy (TF uses this name in https://www.tensorflow.org/api_docs/python/tf/train/replica_device_setter) or PSSpliter, PSBalancer, PSDispatcher?